trailblazer/mip/trending.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+""" Get MIP data for trending."""
+
+from trailblazer.mip import files
+
+
+def parse_mip_analysis(mip_config_raw: dict, qcmetrics_raw: dict, sampleinfo_raw: dict) -> dict:
+    """Parse the output analysis files from MIP for adding info
+    to trend database
+
+    Args:
+        mip_config_raw (dict): raw YAML input from MIP analysis config file
+        qcmetrics_raw (dict): raw YAML input from MIP analysis qc metric file
+        sampleinfo_raw (dict): raw YAML input from MIP analysis qc sample info file
+    Returns:
+        dict: parsed data
+    """
+
+    ### Define output dict
+    outdata = {
+        'analysis_sex': {},
+        'at_dropout': {},
```

Comment:
We could comment this one out

Reply:
This is an important qc-metric to trend as was decided previously. Are you adding this info from some other source?
I think it is bad practise to leave lines of code that are not used in a "commented form". It clutteres the code and makes it more complicated than it is. Removing the unused line is prefered and then adding it in when required. In this case I do not really understand why you want to comment it out as you can just ignore this key when parsing the output in the downstream process (if it is not ready for the "at_dropout"), but I am not up to speed on what you are planning here.

Reply:
Ok, now I see you are just trying to get it to be compatible with the mip5.0 in production, right?

Reply:
... like the title said ;)